documentation/docs/contributors/documentation/running-server.md

```diff
@@ -21,7 +21,7 @@ The scripts below will create a python [virtual environment](https://docs.python
 
     ```sh linenums="1"
     cd documentation
-    python -m venv .venv
+    python3 -m venv .venv
```

Comment:
nit(blocking) how you access python will depend on how you installed python, so I don't think we want to make it specifically python3